scripts/linkerd/benchmark.sh

```diff
@@ -5,16 +5,19 @@ set -euo pipefail
 script_dir=$(dirname ${BASH_SOURCE[0]})
 source "$script_dir/../common.sh"
 
-nr_apps="10"
```

Comment:
Ditto here regarding defaults.

Reply:
Actually, what about having those defaults in single file which can be sourced?

Reply:
Yupp, good idea. Patches welcome :)

Reply:
I've removed the updated defaults from this PR and will issue a separate PR for that. Thank you for calling this out!.